ocean_lib/conftest.py

```diff
@@ -0,0 +1,8 @@
+#  Copyright 2018 Ocean Protocol Foundation
+#  SPDX-License-Identifier: Apache-2.0
+
+# Directory ../tests/ holds many unit tests in its subdirectories.
+# Those tests grab tests/conftest.py::setup_all() which has autouse=True,
+# to get the web3 provider and more. *This* directory needs to access
+# all of those as well. The following import accomplishes that.
+from tests.conftest import setup_all  # noqa: F401
```

Comment:
What is the role for this file?

Reply:
Like the comment says. 

If you have it, you'd have to repeat a whole bunch of code in order to line up things for thorough testing. Which would violate DRY principle. So having this maintains DRY. The comment is there to highlight that it's a pretty big import and why it's there.